src/code/code_800EC960.c

```diff
@@ -1151,336 +1159,319 @@ void func_800EE930(void) {
     D_80131880 = 0;
 }
 
-// DebugAudio_GetOcaInput
-void func_800EE97C(void) {
+void AudioDebug_SetInput(void) { // (not 16-byte-aligned, so this cannot start a new file...)
     Input inputs[4];
     u32 btn;
 
-    PadMgr_RequestPadData(&gPadMgr, &inputs, 0);
+    PadMgr_RequestPadData(&gPadMgr, inputs, 0);
     btn = inputs[3].cur.button;
-    D_8016BAB0 = btn & 0xFFFF;
-    sDebugPadPress = (btn ^ sDebugPadPressRaw) & btn;
-    sDebugPadPressRaw = btn;
+    sDebugPadHold = btn & 0xFFFF;
```

Comment:
```suggestion
    sDebugPadHold = (u16)btn;
```

Reply:
Don't really agree with this, sDebugPadHold is a u32. We're intentionally throwing away bits but not for the purpose of fitting into a smaller data type